PhysicsTools/PatAlgos/python/producersLayer1/metProducer_cfi.py

```diff
@@ -49,7 +49,7 @@
     computeMETSignificance  = cms.bool(False),
     # significance computation parameters, not used
     # if the significance is not computed
-    srcJets = cms.InputTag("selectedPatJets"),
+    srcJets = cms.InputTag("cleanedPatJets"),
```

Comment:
@mmarionncern 

I am wondering what would be the motivation to switch to "cleanedPatJets" from "selectedPatJets" 
and it dose not need to be the same jet collection as we save at the end? 

Regards,
Taejeong